### DISCUSSION TITLE
Unable to sync with server

### DISCUSSION THREAD

**Discussion by jjmrjj79** [openbroadcaster/obplayer#46] [Q&A]
Hi,

Just finished installing server and player. I'm running them both on the same Ubuntu machine, but I'm having issues getting them to sync together. Here's the most current log, with the last few error messages repeating. I've also included pictures of the settings on server and player. Thanks in advance for the help.

[Oct 21 2021 16:34:27 UTC] [error] Traceback (most recent call last):
  File "/usr/share/openbroadcaster/player1/obplayer/obplayer/task.py", line 70, in run
    self.try_run()
  File "/usr/share/openbroadcaster/player1/obplayer/obplayer/scheduler/sync.py", line 134, in try_run
    obplayer.Sync.version_update()
  File "/usr/share/openbroadcaster/player1/obplayer/obplayer/scheduler/sync.py", line 314, in version_update
    version = json.loads(curl_response.buffer)
  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

[Oct 21 2021 16:34:27 UTC] [error] Unable to sync with server.  (invalid id/password/ip combination)
[Oct 21 2021 16:34:27 UTC] [debug] starting thread <SyncShowsThread>
[Oct 21 2021 16:34:27 UTC] [sync] fetching show data from server
[Oct 21 2021 16:34:27 UTC] [debug] starting thread <SyncEmergThread>
[Oct 21 2021 16:34:27 UTC] [debug] starting thread <SyncMediaThread>
[Oct 21 2021 16:34:27 UTC] [debug] starting thread <SyncPlaylogThread>
[Oct 21 2021 16:34:27 UTC] [module] loading module streamer
[Oct 21 2021 16:34:27 UTC] [debug] starting thread <PlayerThread>
[Oct 21 2021 16:34:27 UTC] [error] Unable to sync with server.  (invalid id/password/ip combination)
[Oct 21 2021 16:34:27 UTC] [scheduler] no show found.
[Oct 21 2021 16:34:27 UTC] [player] player has no requests to play; outputting silence
[Oct 21 2021 16:34:27 UTC] [player] now playing track ?: unknown - unknown (id: 0 file:  duration: 3600s type: 'break' source: silence)
[Oct 21 2021 16:34:27 UTC] [debug] audio-break: stopped by execute request
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: audiosrc
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: capsfilter0
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: queue2-0
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: audioconvert0
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: lamemp3enc0
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: queue2-1
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: shout2send
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: audiosrc
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: capsfilter1
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: queue2-2
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: audioconvert1
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: lamemp3enc1
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: queue2-3
[Oct 21 2021 16:34:28 UTC] [debug] adding element to bin: shout2send
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET / HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /extras/select2.min.css HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /common.css HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /extras/jquery.min.js HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /extras/select2.min.js HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /common.js HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /extras/leaflet.css HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /extras/leaflet.js HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /extras/leaflet.draw.js HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /map.js HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /oblogo.svg HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /favicon.ico HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "POST /alerts/geocodes_list HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "POST /alerts/indigenous_languages HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "POST /strings HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /extras/images/marker-icon.png HTTP/1.1" 200 -
[Oct 21 2021 16:34:29 UTC] [debug] 127.0.0.1 "GET /extras/images/marker-shadow.png HTTP/1.1" 200 -
[Oct 21 2021 16:35:27 UTC] [sync] fetching priority broadcast data from server
[Oct 21 2021 16:35:27 UTC] [error] Unable to sync with server.  (invalid id/password/ip combination)
[Oct 21 2021 16:35:57 UTC] [scheduler] no show found.
[Oct 21 2021 16:36:27 UTC] [sync] fetching priority broadcast data from server
[Oct 21 2021 16:36:27 UTC] [sync] fetching show data from server
[Oct 21 2021 16:36:27 UTC] [error] Unable to sync with server.  (invalid id/password/ip combination)
[Oct 21 2021 16:36:27 UTC] [error] Unable to sync with server.  (invalid id/password/ip combination)
![Screenshot from 2021-10-21 13-39-32](https://user-images.githubusercontent.com/91280167/138329749-5861d4f1-7142-474b-bcd2-0e07a72ea570.png)
![Screenshot from 2021-10-21 13-42-21](https://user-images.githubusercontent.com/91280167/138329751-2bc87a69-e829-4458-8181-5b48311f06c1.png)

**Comment by radiorob** [comment]
related to "invalid id/password/ip combination" In the player manager. Clear out the IP Address field. (not really sure what/how this field works, but not needed for basic operation)  Everything else looks correct
